src/main/java/seedu/address/model/person/Nric.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.person;
```

Comment:
Is this class supposed to be in this PR? I think this have been merged from the NRIC PR.

Reply:
Kinda confused as to why this showed up as a change when it should have been merged from the NRIC PR, not sure tbh.